### PR TITLE
fix: tighten up section name from tiles validation

### PR DESCRIPTION
**Pull request by aronshamash**
- We needed to add additional checks for getting section name from tiles arrays

### CODE REVIEW

File: packages/ssr/src/lib/section-from-tiles.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default article =>
+  article &&
+  article.tiles.length > 0 &&
+  article.tiles[0].slices &&
```
Comment (infreezer) [inline packages/ssr/src/lib/section-from-tiles.js]: Is the default value of tiles an empty array
Can we have an article without tiles

Comment (aronshamash) [reply packages/ssr/src/lib/section-from-tiles.js]: maybe yes. I don't know for sure but this will work if it is not present

Comment (infreezer) [reply packages/ssr/src/lib/section-from-tiles.js]: If tiles is undefined .length will throw an error

Comment (aronshamash) [reply packages/ssr/src/lib/section-from-tiles.js]: good point. updated

**Pull request merged**